.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -2,12 +2,17 @@
 name: "beats-auditbeat"
 
 env:
+  BEATS_PROJECT_NAME: "auditbeat"
```

Comment:
What if we load some of the same variable names (from different pipelines) with different values?
Example:
BEATS_PROJECT_NAME: "auditbeat"
BEATS_PROJECT_NAME: "heartbeat"
.....

Reply:
perhaps we have to use:
env:
          BEATS_PROJECT_NAME: $BEATS_PROJECT_NAME
          for pipelines with scripts?